background.ts

```diff
@@ -415,6 +416,18 @@ ipcMainProxy.handle('settings-write', (event, arg) => {
 
 mainEvents.on('settings-write', writeSettings);
 
+ipcMainProxy.handle('transient-settings-fetch', () => {
+  return Promise.resolve(transientSettings);
+});
+
+ipcMainProxy.handle('transient-settings-update', (event, arg) => {
```

Comment:
As this isn't returning a value (and the caller probably doesn't need to wait for the action to finish), this can be `ipcMainProxy.on` instead, right?

Reply:
For some reason,` ipcMainProxy.on` leave untouched the transientSettings value, when I invoke the functionby ipc `emit` from any preferences tab. No errors are thrown...